pkg/controller/queuejobresources/utils.go

```diff
@@ -1,3 +1,33 @@
+/*
+Copyright 2019 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+/*
+Copyright 2019, 2021 The Multi-Cluster App Dispatcher Authors.
```

Comment:
is there a repetition here?

Reply:
I see the two are slightly different.

Reply:
This ok due to new modifications added within this new project in git.